autogen/agentchat/conversable_agent.py

```diff
@@ -78,8 +78,8 @@ def __init__(
         human_input_mode: Optional[str] = "TERMINATE",
         function_map: Optional[Dict[str, Callable]] = None,
         code_execution_config: Union[Dict, Literal[False]] = False,
-        llm_config: Optional[Union[Dict, Literal[False]]] = None,
-        default_auto_reply: Optional[Union[str, Dict, None]] = "",
+        llm_config: Optional[Union[Dict, Literal[False]]] = False,
```

Comment:
Same problem with this change. We probably want to raise the error explicitly when llm_config is omitted for general ConversableAgent.

Reply:
OK, will undo.